releasenotes/notes/deprecating-usm-configuration-values-6c43a0181c2cc821.yaml

```
@@ -0,0 +1,9 @@
+---
+features:
+  - The `network_config.http_replace_rules` configuration has changed to `service_monitoring_config.http_replace_rules`.
+  - The `network_config.max_tracked_http_connections` configuration has changed to `service_monitoring_config.max_tracked_http_connections`.
+  - The `network_config.max_http_stats_buffered` configuration has changed to `service_monitoring_config.max_http_stats_buffered`.
+deprecations:
+  - The `network_config.http_replace_rules` configuration is now deprecated. Use `service_monitoring_config.http_replace_rules` instead.
+  - The `network_config.max_tracked_http_connections` configuration is now deprecated. Use `service_monitoring_config.max_tracked_http_connections` instead.
+  - The `network_config.max_http_stats_buffered` configuration is now deprecated. Use `service_monitoring_config.max_http_stats_buffered` instead.
```

```suggestion
  - The `network_config.http_replace_rules` configuration has been deprecated. Use `service_monitoring_config.http_replace_rules` instead.
  - The `network_config.max_tracked_http_connections` configuration has been deprecated. Use `service_monitoring_config.max_tracked_http_connections` instead.
  - The `network_config.max_http_stats_buffered` configuration has been deprecated. Use `service_monitoring_config.max_http_stats_buffered` instead.
```
Recommend keeping verb tense consistent throughout the release note.

---

thanks, committed
I copied it from a different release note https://github.com/DataDog/datadog-agent/blob/main/releasenotes/notes/enable-http-monitoring-config-value-21ae9f834e3e2c67.yaml